### PR TITLE
[Asset] Thumbnails: add configuration for 1x1 image placeholder, where no image is wanted

### LINKED ISSUE #217: Image Thumbnails Media Queries - no Image
It should be possible to specify no image / thumbnail for a certain media query, 
eg. a teaser element has no image in mobile view.

### PR DESCRIPTION
**Pull request by brusch**
Resolves #217

image

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js [hunk 1]
```diff
@@ -1348,6 +1348,35 @@ pimcore.settings.thumbnail.items = {
...
+            id: myId,
+            style: "margin-top: 10px",
+            border: true,
+            bodyStyle: "padding: 10px;",
+            tbar: this.getTopBar(niceName, myId, panel),
+            html: t("1x1_pixel_placeholder_description"),
+            items: [{
+                xtype: "hidden",
+                name: "type",
+                value: "1x1_pixel"
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js]: value should be "1x1Placeholder" to match with function name to load save config. https://github.com/pimcore/pimcore/blob/bfcce7ca94c08f30fbca69831a13be574c7613d9/bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js#L263

Comment (brusch) [reply bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js]: Good catch, thanks a lot!

Comment (brusch) [reply bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js]: With 8b00edbccb31c90027a42bc7e7354afc6bfcc150 it should be fine now.

**Comment by dvesh3** [comment]
when you set 1x1 pixel placeholder without any media query then it takes height and width, which is wrong:
MicrosoftTeams-image

**Pull request merged**